biocrnpyler/propensities.py

```diff
@@ -166,12 +167,13 @@ def from_dict(cls, propensity_dict):
         merged.update(propensity_dict['species'])
         return cls(**merged)
 
-    def _create_annotation(self, model, **kwargs):
+    def _create_annotation(self, model, propensity_dict_in_sbml, **kwargs):
```

Comment:
I wanted to have the create_annotation function be general and not accept particular arguments like `for_bioscrape` and `propensity_dict_in_sbml`....but I guess this is OK. Maybe I will put this in kwargs as well..and so everything will work like you have set it up.

Reply:
You are right, the ```propensity_dict_in_sbml`` can be part of kwargs, but propensity_dict_in_sbml contains all the SBML names of all parameters and species, which I guess should come handy in all annotation writing process. (I'm just guessing what other annotation processes might need). Feel free to put it to the kwargs though.

Reply:
OK, that's good to know. I will keep it as it is then for now.